File: 389-find-the-difference/389-find-the-difference.c
char findTheDifference(char * s, char * t){
    int sum_S = 0;
    int sum_T = 0;
    int i = 0;
    
    while(t[i]){
        sum_T += t[i];
        if(s[i] != '\0')
            sum_S += s[i];
        i++;
    }
    return (char)(sum_T - sum_S);
}

File: 389-find-the-difference/389-find-the-difference.java
class Solution {
    public char findTheDifference(String s, String t) {
        int ch = 0;
        
        for(int i = 0; i < t.length(); i++){
            if(i != t.length() - 1) ch += t.charAt(i) - s.charAt(i);
            else ch += t.charAt(i);
        }
        return (char)ch;
    }
}